src/main/java/org/folio/helper/PurchaseOrderLineHelper.java

```diff
@@ -896,6 +898,36 @@ private CompletableFuture<Void> processPoLineEncumbrances(CompositePurchaseOrder
     return completedFuture(null);
   }
 
+  private CompletableFuture<Void> updateInventoryItemStatus(CompositePoLine compOrderLine, RequestContext requestContext) {
+    return purchaseOrderLineService.getPoLinesByOrderId(compOrderLine.getPurchaseOrderId(), requestContext)
+      .thenCompose(poLines -> {
+        if (poLines.size() > 1) {
+          List<String> poLinesId = poLines.stream().map(PoLine::getId).collect(toList());
+          return inventoryManager.getItemsByPoLineIdsAndStatus(poLinesId, ItemStatus.ON_ORDER.value(), requestContext)
+            .thenCompose(items -> {
+              if (items.size() > 1) {
+                Optional<JsonObject> poLineItem = items.stream()
+                  .filter(item -> compOrderLine.getId().equals(item.getString("purchaseOrderLineIdentifier"))).findFirst();
+                if (poLineItem.isPresent()) {
+                  JsonObject updatedItem = updateItemStatus(poLineItem.get());
+                  inventoryManager.updateItem(updatedItem, requestContext);
+                }
+              }
+              return CompletableFuture.completedFuture(null);
+            });
+        }
+        return CompletableFuture.completedFuture(null);
+      });
+  }
+
+  private JsonObject updateItemStatus(JsonObject poLineItem) {
```

Comment:
Propose to rename method name to changeItemStatusToOrderClosed or similar to describe purpose of using

Reply:
I think we can use this one: updateItemStatusToClosed